packages/core/src/command/patient-import/patient-import-shared.ts

```diff
@@ -0,0 +1,127 @@
+import {
+  normalizeDate,
+  normalizeEmailStrict,
+  normalizeExternalId,
+  normalizeGender,
+  normalizePhoneNumberStrict,
+  normalizeState,
+  normalizeZipCodeNew,
+  PatientImportPatient,
+  toTitleCase,
+} from "@metriport/shared";
+import { PatientPayload } from "./patient-import";
+
+const globalPrefix = "patient-import";
+
+export type FileStages = "raw" | "valid" | "invalid";
+
+function createCxJobPrefix(cxId: string, jobId: string): string {
+  return `cxid=${cxId}/jobid=${jobId}`;
+}
+
+function createFilePathPatients(cxId: string, jobId: string, patientId: string): string {
+  return `${createCxJobPrefix(cxId, jobId)}/patients/patientid=${patientId}/status.json`;
+}
+
+function createFilePathFiles(cxId: string, jobId: string, stage: FileStages): string {
+  return `${createCxJobPrefix(cxId, jobId)}/files/${stage}.csv`;
+}
+
+export function createFileKeyJob(cxId: string, jobId: string): string {
+  return `${globalPrefix}/${createCxJobPrefix(cxId, jobId)}/status.json`;
+}
+
+export function createFileKeyPatient(cxId: string, jobId: string, patientId: string): string {
+  const fileName = createFilePathPatients(cxId, jobId, patientId);
+  const key = `${globalPrefix}/${fileName}`;
+  return key;
+}
+
+export function createFileKeyFiles(cxId: string, jobId: string, stage: FileStages): string {
+  const fileName = createFilePathFiles(cxId, jobId, stage);
+  const key = `${globalPrefix}/${fileName}`;
+  return key;
+}
+
+export const patientImportCsvHeaders = [
+  "externalid",
+  "firstname",
+  "lastname",
+  "dob",
+  "gender",
+  "zip",
+  "city",
+  "state",
+  "addressline1",
+  "addressline2",
+  "phone1",
+  "email1",
+  // "phone2",
+  // "email2",
```

Comment:
We need to decide what to do here, in case one of the next two bulk imports use those "two" columns

Reply:
Let's make sure all runs now just use `phone1` and `email1` then extended in v1.

Reply:
Done for first 2 imports.